src/datasets/packaged_modules/json/json.py

```diff
@@ -80,8 +80,11 @@ def _generate_tables(self, files):
                     pa_table = paj.read_json(
                         file,
                         read_options=self.config.pa_read_options,
-                        parse_options=self.config.pa_parse_options,
+                        # parse_options=self.config.pa_parse_options,
```

Comment:
why not use the parse options ?

Reply:
To avoid breaking changes I'd still keep the parse options but only for the `newlines_in_values` parameter, since the schema is now used for casting the table to allow int<->str conversions.

Reply:
That is what I finally did!!! ;)